docs/source/markdown/podman-images.1.md

```diff
@@ -52,6 +52,10 @@ Filter output based on conditions provided
 Change the default output format.  This can be of a supported type like 'json'
 or a Go template.
 
+**--history**
+
+Display the image name history
```

Comment:
I would love it to be a bit more verbose. What is an image name history? When can it be useful?

Reply:
Yes I agree, I added a more verbose comment.